buildkitd/Earthfile

```diff
@@ -1,6 +1,6 @@
 
 buildkitd:
-    ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:earthly-master+build
+    ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:vlad/exporters+build
```

Comment:
can this be switched back now that the buildkit PR was merged?

Reply:
Yes done